src/renderer/components/dialogs/Settings.js

```diff
@@ -93,26 +103,37 @@ class Settings extends React.Component {
     this.props.userFeedback({ type: 'success', text: 'Account updated successfully' })
   }
 
+  onAdvancedSettingsOpen () {
+    ipcRenderer.send('retrieveAdvancedSettingsObj')
+  }
+
+  retrieveAdvancedSettingsObjResp (e, advancedSettingsObj) {
```

Comment:
Why not name this `advancedSettings`?

Reply:
advancedSettings seems to be too generic to me. It's not really explaining what this method is there for.